### PR TITLE
feat: add support for GraphQL endpoint

### PR DESCRIPTION
**Pull request by imprvhub**
-Add GraphQL method to CrowdinClient for interfacing with the GraphQL endpoint.
-The implementation supports query strings and optional variables.
-Includes comprehensive test coverage for request validation.

Implements #180

### CODE REVIEW

**Comment by imprvhub** [comment]
@andrii-bodnar PR ready for review!

**Comment by andrii-bodnar** [comment]
@imprvhub thanks for the contribution!

Please add documentation about this endpoint to the Readme file.

As a reference, see the [JS API Client Readme](https://github.com/crowdin/crowdin-api-client-js?tab=readme-ov-file#graphql-api).

**Comment by imprvhub** [comment]
@andrii-bodnar As requested, I have included the GraphQL API documentation in the README as part of the implementation commit [fc0648f](https://github.com/crowdin/crowdin-api-client-python/pull/189/commits/fc0648fcbb84cbc4739af4ca2b0e909f880c319d).

File: README.md [hunk 1]
```diff
@@ -159,6 +159,30 @@ class FirstCrowdinClient(CrowdinClient):
     EXTENDED_REQUEST_PARAMS = {"proxies": proxies}
 ```
 
+### GraphQL API
+This library also provides possibility to use [GraphQL API](https://developer.crowdin.com/graphql-api/) (only for Crowdin Enterprise).
```
Comment (andrii-bodnar) [inline README.md:163]:
```suggestion
### GraphQL API

This library also provides the possibility to use [GraphQL API](https://developer.crowdin.com/graphql-api/):
```

**Review by andrii-bodnar — CHANGES_REQUESTED** [review]

**Pull request merged**